tgui/packages/tgui/interfaces/PlayerTicketHistory.tsx

```diff
@@ -108,9 +108,8 @@ const TicketHistory = (props: any) => {
     );
   }
 
-  const [activeTicket, setActiveTicket] = useLocalState<TicketData | undefined>(
-    'ticket',
-    undefined,
+  const [activeTicket, setActiveTicket] = useState<TicketData | undefined>(
+    undefined
   );
```

Comment:
```suggestion
  const [activeTicket, setActiveTicket] = useState<TicketData | undefined>();
```
just cleaner - the only problem here is that now, to reset its state, you must use `setActiveTicket(undefined)` not `setActiveTicket()`

Reply:
That function is only called here:

https://github.com/tgstation/tgstation/blob/93b316690ea04e4d69cf739f5d314cd9a92f4630/tgui/packages/tgui/interfaces/PlayerTicketHistory.tsx#L131-L139